install/tools/ipactl

```diff
@@ -138,7 +138,8 @@ def version_check():
     try:
         installutils.check_version()
     except (installutils.UpgradeMissingVersionError,
-            installutils.UpgradeDataOlderVersionError):
+            installutils.UpgradeDataOlderVersionError) as exc:
+        emit_err("IPA version error: %s" % exc)
```

Comment:
This is an attempt to be consistent within the function. In some other places of the file, there are also str(e) usages... I'm either keeping it this way or I'm fixing the entire file. Which would you prefer?